docs/starter/quick-start.md

```diff
@@ -1,48 +1,64 @@
 # PWA Starter Quick Start
 
-## Prerequisites
+You can started with the PWA Starter by using the PWABuilder CLI, which allows you to create a fresh, templated progressive web app projects in just moments.
 
-There are just a few things you will need before getting started:
+There are just a couple things you will need before getting started:
 
-- A Github Account, which you can create at [GitHub](https://github.com/).
 - A code editor, such as [Visual Studio Code](https://code.visualstudio.com/).
-- A web browser, such as [Microsoft Edge](https://www.microsoft.com/en-us/edge).
-- [Node.js](https://docs.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-windows)
-- [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- [Node.js and npm](https://docs.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-windows)
 
-## Start a New App
+## Installing the CLI
```

Comment:
Should we add something about running the cli without installing it, like with npx?